app/Http/Middleware/CheckIfUserBanned.php

```diff
@@ -21,6 +22,9 @@ public function handle(Request $request, Closure $next)
             return $next($request);
         }
 
+        $userLogType = session()->get('kanka.userLog', UserLog::TYPE_BANNED_LOGIN);
```

Comment:
I don't think this is correct: this will trigger a log when a banned user refreshes the page for the first being after being banned, which isn't useful information. Instead it could be a new key `TYPE_BANNED_LOGOUT`